music21/musicxml/m21ToXml.py

```diff
@@ -4799,14 +4888,6 @@ def articulationToXmlTechnical(self, articulationMark):
         >>> mxOther = MEX.articulationToXmlTechnical(g)
         >>> MEX.dump(mxOther)
         <other-technical>unda maris</other-technical>
-
-        Same with technical marks not yet supported.
-        TODO: support HammerOn, PullOff, Bend, Hole, Arrow.
-
-        >>> h = articulations.HammerOn()
-        >>> mxOther = MEX.articulationToXmlTechnical(h)
-        >>> MEX.dump(mxOther)
-        <other-technical />
```

Comment:
If Bend, Hole, Arrow are still not supported, then we should leave this but just change the example to one of those.